.github/workflows/compile.yml

```
@@ -0,0 +1,23 @@
+name: Compile Latex 
+
+
+on:
+  pull_request:
+    branches: [ main, master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)